### ISSUE TITLE
Should we be able to "link" other code?

### ISSUE THREAD

**Issue by ubsan** [leios/algorithm-archive#69]
For example, in [convolutions][conv], the FFT code is out of date. Should we be able to link in other code? Preferably with checks for when the API breaks?

**Comment by leios** [comment]
Oh, I see what you mean. Yeah, this is a good point of discussion.

I would say that we should link to previous code (if only to point the reader in the direction for more information); however, in the case of fft's I feel it's probably better to use a standard fft library (like fftw), because that's what people will see in practice. If there is no good library to perform the operation already, then we should link to previous code... And I'm not sure the best way to do that.

**Comment by jiegillet** [comment]
I don't think we should link to other code, I like each chapter to be independent. Whenever relevant,  we should link to other AAA chapters, but not rely on specific code.
